src/main/java/seedu/address/logic/commands/ImportVolunteerCsvCommand.java

```diff
@@ -106,7 +109,7 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
                         argMultimap.getValue(PREFIX_ADDRESS).get());
                 Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
 
-                Volunteer volunteer = new Volunteer(name, gender, birthday, phone, email, address, tagList);
+                Volunteer volunteer = new Volunteer(name, volunteerId, gender, birthday, phone, email, address, tagList);
```

Comment:
Might want to convert volunteerId to upper case too for string comparison later on! Easier to ensure that everything is in caps!

Reply:
Just checking, if I invoked toUpperCase when constructing VolunteerId in the VolunteerId class, do i still have to convert volunteerId to upper case here?

Reply:
Oh, don't have to. If it's done in volunteerID itself then no need to touch it anymore. I might have missed it!